stdlib/Random/src/normal.jl

```diff
@@ -47,7 +50,7 @@ julia> randn(rng, ComplexF32, (2, 3))
         r = rand(rng, UInt52Raw())
 
         # the following code is identical to the one in `_randn(rng::AbstractRNG, r::UInt64)`
-        r &= 0x000fffffffffffff
+        r = _take52(rng, r)
```

Comment:
This is not correct, `rand(rng, UInt52Raw())` generates 52 random bits in low position, and actually you already defined a method for it which discards the low bits of a random `UInt64`. So by calling `_take52` here you discard 12 more. I suggest to just write `r = rand(rng, UInt52())` which takes care of giving good 52 bits which 0 high bits.